pkg/util/workqueue/queue.go

```diff
@@ -113,6 +113,16 @@ func (q *Type) Get() (item interface{}, shutdown bool) {
 	return item, false
 }
 
+// Get the head of the queue
+func (q *Type) Head() interface{} {
```

Comment:
A method that returns the first element of a queue but doesn't remove it is usually called Peek.